cmd/cli/main.go

```diff
@@ -109,8 +110,8 @@ VERSION:
 	log        = logger.GetOrCreate("mx-sdk-go/cmd/cli")
 )
 
-var HOME = os.Getenv("HOME")
-var pathGeneratedWallets = HOME + "/MultiversX/testnet/filegen/output/walletKey.pem"
+var homePath = os.Getenv("HOME")
+var pathGeneratedWallets = homePath + "/MultiversX/testnet/filegen/output/walletKey.pem"
```

Comment:
will this work on OSX or windows with of.Getenv("HOME")? 
The path "MultiversX/testnet/filegen/output/walletKey.pem" is very specific and will this be generated always? 
Is there no way to generate a walletKey.pem in the current folder?

Reply:
This does not come from this PR actually...

Reply:
This is specific to the local system testnet. That will almost certain run only on linux.